sdk/storage/storage-file/src/highlevel.common.ts

```diff
@@ -7,7 +7,7 @@ import { FileHTTPHeaders, Metadata } from "./models";
 import { Aborter } from "./Aborter";
 
 /**
- * Option interface for uploadFileToAzureFile and uploadSeekableStreamToAzureFile.
+ * Option interface for FileClient.uploadFile() and FileClient.uploadSeekableStream().
```

Comment:
We should remove highlevel.common.ts too after removing highlevel.node/browser.ts. Or at least have some better name. Sounds strange to have a "common", because you already move all browser/node functions into `FileClient.ts`

Reply:
Yeah agree. Removed.

Reply:
The option interfaces were moved into clients. I didn't rename them though.  Some of the options are used by multiple functions so it doesn't make sense to follow the convention of having the function name in the option interface name.